### ISSUE TITLE
Check Dropdown doesn't close when opened until Github Desktop is restarted

### ISSUE THREAD

**Issue by faseehlodhi** [desktop/desktop#19413]
### The problem

When checking the branch checks by clicking on the "checks" icon, a drop-down menu appears as expected. However, the drop-down does not dismiss and remains visible until the entire application is restarted. Even after clicking the arrow or the "checks" icon again, the drop-down remains on display.

### Release version

3.4.6

### Operating system

macOS Sonoma 14.7

### Steps to reproduce the behavior

1. Open the application and navigate to a branch.
2. Click on the "checks" icon to view the branch checks.
3. Notice that the drop-down menu appears as expected.
4. Attempt to dismiss the drop-down by clicking the arrow or the "checks" icon again.

**Comment by steveward** [comment]
Thanks for the report @faseehlodhi. I just testing this out and was unable to reproduce the issue. Are you able to record a short video clip of this behavior so I can confirm there isn't something I'm missing? Thanks!

**Comment by faseehlodhi** [comment]
> Thanks for the report @faseehlodhi. I just testing this out and was unable to reproduce the issue. Are you able to record a short video clip of this behavior so I can confirm there isn't something I'm missing? Thanks!

@steveward Thanks for looking into it, I have attached the video, let me know if it's reproducible, or something on my end, nothing serious but just felt like reporting.

https://github.com/user-attachments/assets/aa7e8d9a-8a8f-40db-8849-8896fc6a7e52

**Comment by steveward** [comment]
Thanks, I was able to reproduce the issue now. Clicking outside of the checks modal did close it -- does that work for you? I'll mark this as a bug for us to investigate.

**Comment by faseehlodhi** [comment]
> Thanks, I was able to reproduce the issue now. Clicking outside of the checks modal did close it -- does that work for you? I'll mark this as a bug for us to investigate.

Yes it works for me when I click outside of the check, I just checked although it didn't work for me before

**Comment by gilluminate** [comment]
I've been experiencing this as well. Using the `escape` key is also an interim solution.

**Comment by Cedric-Ab** [comment]
for me:
It doesn't close, if you press something in the menu band, even though it should.
